tests/bpftrace.cpp

```diff
@@ -1021,48 +1021,19 @@ TEST(bpftrace, sort_by_key_int)
   EXPECT_THAT(values_by_key, ContainerEq(expected_values));
 }
 
-TEST(bpftrace, sort_by_key_int_int)
```

Comment:
Sorting by multiple keys is now broken, right? Is that expected?

Reply:
I think I forgot about this as I was deleting. Let me fix.

Reply:
Ok, re-added. Thanks for catching this.